.github/workflows/stable-release.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - 'main'
+      - 'v*'
+
+    paths: 
+      - 'STABLE-RELEASE'
+
+name: KubeArmor stable release 
+
+jobs:
+  build:
+    name: Create KubeArmor Release - 18.04
+    runs-on: ubuntu-18.04
+    timeout-minutes: 20
+    steps:
+      - name: Checkout KubeArmor code
+        uses: actions/checkout@v2
+
+      - name: Pull Docker Hub image
+        run: |
+            value=`cat STABLE-RELEASE`
+            docker pull docker.io/kubearmor/kubearmor:$value
+            docker image tag kubearmor/kubearmor:$value kubearmor/kubearmor:stable
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_AUTHTOK }}
+
+      - name: Push KubeArmor images to Docker
```

Comment:
I think we do not need to explicitly push ... `docker image tag kubearmor/kubearmor:$value kubearmor/kubearmor:stable` will take care of creating the tag on the docker repo.